File: examples/FaxDeleteExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
end

begin
    Dropbox::Sign::FaxApi.new.fax_delete(
        "fa5c8a0b0f492d768749333ad6fcc214c111e967", # fax_id
    )
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling FaxApi#fax_delete: #{e}"
end

File: examples/FaxDeleteExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
$config->setUsername("YOUR_API_KEY");

try {
    (new Dropbox\Sign\Api\FaxApi(config: $config))->faxDelete(
        fax_id: "fa5c8a0b0f492d768749333ad6fcc214c111e967",
    );
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling FaxApi#faxDelete: {$e->getMessage()}";
}
